CME/admin/components/FrontMatter/Edit.php

```diff
@@ -114,7 +112,8 @@ protected function setDefaultValues()
 			);
 
 			$default_provider_id = SwatDB::queryOne($this->app->db, $sql);
-			$this->ui->getWidget('provider')->value = $default_provider_id;
+			$this->ui->getWidget('providers')->values =
+				array($default_provider_id);
```

Comment:
This should be restructured to allow multiple default providers. `getDefaultProviderShortname()` could become `getDefaultProviderShortnames()` and the query above could return and array.